locals.tf

```
@@ -0,0 +1,6 @@
+locals {
+  default_tags = {
+    scope      = "terraform-managed"
+    repository = "jenkins-infra_digitalocean"
```

```suggestion
    repository = "jenkins-infra/digitalocean"
```

---

```suggestion
    jenkins_infra_repository = "digitalocean"
```

Since / are forbidden in DO tags